src/core/world_state.c

```diff
@@ -0,0 +1,2 @@
+
+#include "world_state.h"
```

Comment:
Line 1 is empty.

Also, this is another empty file that introduces an extra translation unit.